docs/UserGuide.md

```diff
@@ -79,13 +79,13 @@ This section introduces ...
 
 ## Commands
 
-This section introduces the various commands used in Nuudle.
+The command is the main way to schedule appointment for a patient with Nuudle.
 
-[general commands](#general-commands), [patient management](#patient-management), [command summary](#command-summary)
+This section provides information about the commands on [general commands](#general-commands), [patient management](#patient-management), [command summary](#command-summary) accordingly.
```

Comment:
Missed out on the `appointment-management` here.

Reply:
thanks for the spot!